app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -460,5 +461,39 @@ def parse_image_name(image, image_ref)
         },
       ]
     end
+
+    def parse_volumes(entity)
+      volumes = entity.spec.volumes
+      volumes.collect do |volume|
+        {
+          :type                    => 'ContainerVolumeKubernetes',
+          :name                    => volume.name,
+          :empty_dir_medium_type   => volume.emptyDir.try(:medium),
+          :gce_pd_name             => volume.gcePersistentDisk.try(:pdName),
+          :git_repository          => volume.gitRepo.try(:gitRepository),
+          :git_revision            => volume.gitRepo.try(:git_revision),
+          :nfs_server              => volume.nfs.try(:server),
+          :iscsi_target_portal     => volume.iscsi.try(:targetPortal),
+          :iscsi_iqn               => volume.iscsi.try(:iqn),
+          :iscsi_lun               => volume.iscsi.try(:lun),
+          :glusterfs_endpoint_name => volume.glusterfs.try(:endpointsName),
+          :claim_name              => volume.persistentVolumeClaim.try(:claimName),
+          :rbd_ceph_monitors       => volume.rbd.try(:cephMonitors).to_a.join(','),
+          :rbd_image               => volume.rbd.try(:rbdImage),
+          :rbd_pool                => volume.rbd.try(:rbdPool),
+          :rbd_rados_user          => volume.rbd.try(:radosUser),
+          :rbd_keyring             => volume.rbd.try(:keyring),
+          :common_path             => [volume.hostPath.try(:path), volume.nfs.try(:path), volume.glusterfs.try(:path)].compact.first,
```

Comment:
@zakiva can you check if this notation works? (and it doesn't give rubocop warnings)

          :common_path             => volume.hostPath.try(:path) ||
                                      volume.nfs.try(:path) ||
                                      volume.glusterfs.try(:path),

It seems more readable to me. (Same for all other entries below)